### ISSUE TITLE
E0121 false positive

### ISSUE THREAD

**Issue by matklad** [intellij-rust/intellij-rust#1954]
```Rust
    let two_arg : Vec<(&str, fn(_, _) -> _)>= vec![
        ("layer", CallKind::Layer),
        ("with_skip", CallKind::WithSkip)
    ];
```
